tests/integration/block_createBlock_api_test.go

```diff
@@ -1,159 +0,0 @@
-/*
```

Comment:
will we see much of a integration test speed up by removing these tests that took the API path?

Reply:
no, i haven't noticed or expect a speedup of any consequence